src/chericat.c

```diff
@@ -99,11 +122,25 @@ main(int argc, char *argv[])
 		return 0;
 	}
 
-	sqlite3 *db = NULL;
-
 	static int debug_level = 0;
 	set_print_level(debug_level);
 
+#ifdef HAVE_SIGACTION
+	{
```

Comment:
Stray bracket? This ifdef branch probably doesn't compile.

Reply:
hmm.. not sure what happened, it did compile when I tested, perhaps fat fingers somehow before I commit. Added the } back on.

Reply:
There doesn't seem to be anything defining `HAVE_SIGACTION` so I think the code was never compiled

Reply:
fixed